src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -6493,7 +6499,7 @@ private static string QuoteIdentifier(ReadOnlySpan<string> strings)
 
             // Stitching back together is a little tricky. Assume we want to build a full multi-part name
             //  with all parts except trimming separators for leading empty names (null or empty strings,
-            //  but not whitespace). Separators in the middle should be added, even if the name part is 
+            //  but not whitespace). Separators in the middle should be added, even if the name part is
```

Comment:
(These whitespace-related changes are there because I have my editor set to remove trailing newlines on save. I can revert those hunks if you feel like it)